Update component “breadcrumbs” (#124)
Zywave/booster: src/_areas/design-system/components/breadcrumbs.md

<hr>

## Types/States

### Default

Default state is the shortest form of a Breadcrumb navigation. It contains the current page with one prior Breadcrumb.

![Breadcrumbs - default](/images/default.png)

<br>

### Long Breadcrumb trail

When there are 5 or more pages in a Breadcrumb trail, we recommend using an ellipsis to hide everything between the first page and the second to last page. Clicking the ellipsis will expand to reveal all the hidden Breadcrumbs.

![Breadcrumbs - long](/images/long.png)

<br>

### Long Breadcrumb trail - expanded

When the ellipsis is clicked, the Breadcrumb trail will expand to show all hidden pages. When the Breadcrumbs are fully expanded, any overflow will wrap to a new line.

> ![Breadcrumbs - long expanded](/images/long-–-expanded.png)

Reviewer: This entire section makes more sense to be part of the Behavior section because it is explaining how breadcrumbs behaves and not necessarily the types or state of a breadcrumb.

Author: That makes sense. I have relocated them to the behavior section!

Reviewer: Beautiful! This is good to go!

<hr>

## Best Practices

> ### Page title length

Reviewer: ```suggestion
### Breadcrumb title length
```

* We recommend keeping Breadcrumb titles to 30 characters or less, especially in longer trails, but leave the length up to the consumer

> * Page titles will truncate after 30 characters with an ellipsis at the end

Reviewer: ```suggestion
* Breadcrumb titles will truncate after 30 characters with an ellipsis at the end
```

![truncated title breadcrumb](/images/truncate.png)

<docs-spacer>

</docs-spacer>

### Separators should mimic movement

> Use chevrons as separators between page/Breadcrumb titles. They not only help separate between different page/Breadcrumb titles, but also help indicate hierarchy.

Reviewer: ```suggestion
Use chevrons as separators between Breadcrumb titles. They not only help separate between different Breadcrumb titles, but also help indicate hierarchy.
```

<docs-grid columns="2">

<div>

![Always use right arrows in breadcrumb trials](/images/1-1.png)

<docs-do>

> Always use right arrows in breadcrumb trials.

Reviewer: ```suggestion
Always use right arrows in breadcrumb trails.
```

</docs-do>

</div>

<div>

![Avoid using icons that are not right arrows, such as forward slash "/", dash "-", and etc.](/images/1-2.png)

<docs-do-not>
Avoid using icons that are not right arrows, such as forward slash "/", dash "-", and etc.
</docs-do-not>

</div>

</docs-grid>

<docs-spacer>

</docs-spacer>

### The first Breadcrumb should always be spelled out

When placing ellipses, always leave the first item visible and avoid using ellipses at the beginning.

<docs-grid columns="2">

<div>

> ![Consider using ellipses from the second item. Always leave the first item so users can easier find where the structure starts](/images/2-1.png)
> 
> <docs-do>
> Consider using ellipses from the second item. Always leave the first item so users can easier find where the structure starts.

Reviewer: ```suggestion
![Consider using ellipses from the second item onwards. Always leave the first item visible so users can easily find where the structure starts.](/images/2-1.png)

<docs-do>
Consider using ellipses from the second item onwards. Always leave the first item visible so users can easily find where the structure starts.
```
